prototypes/output_slice/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(output_slice_parallel output_slice_parallel.cc)
+deal_ii_setup_target(output_slice_parallel)
```

Comment:
If the prototype is called output_slice_parallel I would call the folder in the same way so that things are easier to understand :)